File: samples/snippets/csharp/VS_Snippets_VBCSharp/VbPowerPacksShapeScale/CS/ShapeScale.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace VbPowerPacksShapeScaleCS
{
    public partial class ShapeScale : Form
    {
        public ShapeScale()
        {
            InitializeComponent();
        }

        // <Snippet1>
        private void ovalShape1_Click(System.Object sender, System.EventArgs e)
        {
            bool state = true;
            if (state == false)
            {
                ovalShape1.Scale(new SizeF(2, 3));
                state = true;
            }
            else
            {
                ovalShape1.Scale(new SizeF(0.5f, 0.333f));
                state = false;
            }
        }
        // </Snippet1>
    }
}

File: samples/snippets/visualbasic/VS_Snippets_VBCSharp/VbPowerPacksShapeScale/VB/ShapeScale.vb
Public Class ShapeScale
    ' <Snippet1>
    Private Sub OvalShape1_Click() Handles OvalShape1.Click
        Dim state As Boolean
        If state = False Then
            OvalShape1.Scale(New SizeF(2, 3))
            state = True
        Else
            OvalShape1.Scale(New SizeF(0.5, 0.333))
            state = False
        End If
    End Sub
    ' </Snippet1>
End Class
